packages/ckeditor5-dev-env/lib/release-tools/utils/transform-commit/transform-commit-utils.js

```diff
@@ -39,31 +39,100 @@ const transformCommitUtils = {
 	},
 
 	/**
-	 * Changes user's name to link that leads to the user's profile.
+	 * Replaces reference to user (@name) with a link to his profile.
 	 *
-	 * @param {String} sentence
+	 * @param {String} comment
 	 * @returns {String}
 	 */
-	linkGithubUsers( sentence ) {
-		return sentence.replace( /@([\w\d_-]+)/g, '[@$1](https://github.com/$1)' );
+	linkToGithubUser( comment ) {
+		return comment.replace( /@([0-9A-Z_-]+\/?)/ig, ( matchText, nickName ) => {
+			if ( nickName.endsWith( '/' ) ) {
+				return matchText;
+			}
+
+			return `[@${ nickName }](https://github.com/${ nickName })`;
+		} );
 	},
 
 	/**
-	 * Changes references to issue to links that lead to the GitHub issue page.
+	 * Replaces reference to issue (#ID) with a link to the issue.
 	 *
-	 * @param {String} sentence
+	 * @param {String} comment
 	 * @returns {String}
 	 */
-	linkGithubIssues( sentence ) {
+	linkToGithubIssue( comment ) {
 		const packageJson = getPackageJson();
 		const issuesUrl = ( typeof packageJson.bugs === 'object' ) ? packageJson.bugs.url : packageJson.bugs;
 
 		if ( !issuesUrl ) {
 			throw new Error( `The package.json for "${ packageJson.name }" must contain the "bugs" property.` );
 		}
 
-		return sentence.replace( /#([0-9]+)/g, ( _, issueId ) => {
-			return `[#${ issueId }](${ issuesUrl }/${ issueId })`;
+		return comment.replace( /(.?)#([0-9]+)/g, ( matchText, charBeforeIssue, issueId ) => {
+			// Don't replace anything if the '#ID' belongs to another part of the comment.
+			if ( charBeforeIssue && /[A-Z0-9_-]/i.test( charBeforeIssue ) ) {
```

Comment:
You can put this regexp in the first one – just replace `(.?)` with it.

Reply:
As above, it does not help. I had to leave it.